apps/internal/base/base.go

```diff
@@ -202,8 +203,13 @@ func (b Client) AcquireTokenSilent(ctx context.Context, silent AcquireTokenSilen
 	authParams.HomeaccountID = silent.Account.HomeAccountID
 
 	if s, ok := b.manager.(cache.Serializer); ok {
-		b.cacheAccessor.Replace(s)
-		defer b.cacheAccessor.Export(s)
+		var isAppCache bool
+		if silent.Account.IsZero() {
+			isAppCache = true
+		}
```

Comment:
is this only available to cca? not sure as it's in base.go

Reply:
Thanks for this, moved it to confidential.go in the latest commit : https://github.com/AzureAD/microsoft-authentication-library-for-go/pull/184/commits/37a493c54450f34ef29b42c6252c7989b05b45e4